app/models/user.rb

```diff
@@ -1,6 +1,6 @@
 class User < ApplicationRecord
   has_secure_password
-  validates_presence_of :password_confirmation, :password, :email, :name
+  validates_presence_of :password, :email, :name
```

Comment:
Does the confirmation still work?

Reply:
Yep, there is still a validates_confirmation_of :password on the model